### PR TITLE
Improve inactive issues function

### LINKED ISSUE #4: Improve the alert process of inactive issues
The inactive issue function should be improved/splitted into several functions:

* Default checks
    * [x] Do no watch issues with `work in progress` or `PR in progress` or `mute-bot` label
* If the issue is in the current milestone
    * [x] Make sure someone is still working on the issue (send an alert after 14 days without comment)
* If the issue is in a future milestone
    * [x] Do nothing
* If issue is not in a milestone
    * [x] Send an alert after 7 days of inactivity (and close after 14 days)

### PR DESCRIPTION
**Pull request by Odonno**
fix #4

### CODE REVIEW

File: inactiveIssues/index.ts [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+                        });
+
+                    const issuesInTheCurrentMilestone = issuesToCheck
+                        .filter(issue => issue.milestone && issue.milestone.number === currentMilestone.number);
+
+                    const issuesNotInMilestone = issuesToCheck
+                        .filter(issue => issue.milestone && issue.milestone.number < currentMilestone.number);
+
+                    const inactiveIssuesInTheCurrentMilestone = issuesInTheCurrentMilestone.filter(issue => {
+                        return detectIssueWithoutActivity(issue, 14);
```
Comment (nmetulev) [inline inactiveIssues/index.ts]: Should this be NUMBER_OF_DAYS_WITHOUT_ACTIVITY * 2?

File: inactiveIssues/index.ts [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+                        })
+                        .filter(issue => {
+                            return !isIssueContainsExclusiveLabels(issue, exclusiveLabels);
+                        });
+
+                    const issuesInTheCurrentMilestone = issuesToCheck
+                        .filter(issue => issue.milestone && issue.milestone.number === currentMilestone.number);
+
+                    const issuesNotInMilestone = issuesToCheck
+                        .filter(issue => issue.milestone && issue.milestone.number < currentMilestone.number);
```
Comment (nmetulev) [inline inactiveIssues/index.ts]: I might have missed it, it doesn't seem to check for issues that don't have milestone.

Comment (Odonno) [reply inactiveIssues/index.ts]: Oh, good catch. Should be `!issue.milestone || issue.milestone.number < currentMilestone.number`

File: inactiveIssues/index.ts (hunk 1 above)
Comment (Odonno) [reply inactiveIssues/index.ts]: I think so.

**Comment by nmetulev** [comment]
There seems to be a conflict? Feel free to merge after you resolve it :)

**Pull request merged**